codice-itest-autoconfigure/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.codice</groupId>
+        <artifactId>codice-itest</artifactId>
+        <version>3.0.0</version>
+    </parent>
+
+    <artifactId>codice-itest-autoconfigure</artifactId>
+    <name>Codice :: ITest :: Autoconfigure</name>
+    <description>Spring autoconfiguration module for Codice ITest</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-autoconfigure</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-configuration-processor</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.codice</groupId>
+            <artifactId>codice-itest-api</artifactId>
+            <version>${project.version}</version>
```

Comment:
❓ In different POMs codice-itest-api has a version of project.version.  Should the version be an independent value different from the project version?  Also is there an codice-itest-api version of 3.0.0?

Reply:
yeah, ${project.version} is defined in the parent pom, so all of the modules - api, impl, autoconfigure, starter, archetype - have the same version.